script/add-page-map.ts

```diff
@@ -0,0 +1,50 @@
+import fetch from 'node-fetch';
+import path from 'path';
+import argv from 'yargs';
+import { readFile, writeFile } from '../src/helpers/fileUtils';
+
+(global as any).fetch = fetch;
+
+// requires a JSON file with three keys: moduleTitle, moduleID, canonicalID
+const addPageMap = () => {
+  const {
+    mapPath,
+    bookSlug,
+    bookTitle,
+    bookId,
+    canonicalBookTitle,
+    canonicalBookId,
+  } = argv.string('mapPath').argv as any as {
+    mapPath: string,
+    bookSlug: string,
+    bookTitle: string,
+    bookId: string,
+    canonicalBookTitle: string,
+    canonicalBookId: string,
+  };
+
+  if (!mapPath || !bookSlug || !bookId || !canonicalBookId) {
```

Comment:
Maybe `bookTitle` and `canonicalBookTitle` should be required?

Reply:
I think it's ok - they're not required for the map to work and you can always look them up.